packages/ui/theme-builder/app/theme-builder.tsx

```diff
@@ -8,11 +8,11 @@ import { generateColors, getPreviewStyles } from './generate-colors';
 import { ThemeDialog } from './theme-dialog';
 import { ToggleGroup } from './toggle-group';
 
-const lightAccentDefault = '#2F3037';
+const lightAccentDefault = '#6C47FF';
```

Comment:
I would assume we'd not want our default accent color to be the clerk purple and default to the current gray.

Reply:
No worries on the delay!

I went with Alvish's recommendation for now to help distinguish between the gray/accent (which I agree has proven tricky to differentiate at times)